### ISSUE TITLE
Apps are not natively supported by Windows 10. 

### ISSUE THREAD

**Issue by kolbasa** [neutralinojs/neutralinojs#959]
**Describe the bug**
Apps cannot be used with a fresh Windows 10 installation.
Only an empty window opens. No error message, no log messages.
The problem: Windows 10 does not come bundled with WebView2.
Even if all Windows updates are installed.

**To Reproduce**
1. Create a fresh Windows 10 Install
2. Open a simple Neutralinojs App

**Expected behavior**
Ideally, the first launch would prompt the user to install WebView2.
Or at least there should be a meaningful error message.
Other similar frameworks install WebView2 automatically (e.g. Tauri).

**Screenshots/Screen recordings**
![Bildschirmfoto 2022-07-20 um 11 24 01](https://user-images.githubusercontent.com/3901582/180023345-e368f069-bdbf-4916-b608-8784be67ed38.png)

**Specifications**  
-OS: Windows 10 x64
-Neutralinojs version: v4.6.0
-Neutralinojs client library version: v3.5.0
-Neutralinojs CLI version: v9.3.1

**Comment by ZaDarkSide** [comment]
Yeah same here, it should at least prompt the user to install WebView2

**Comment by shalithasuranga** [comment]
Closing this since now Neutralinojs displays a message if webview2 is not installed. Thanks for reporting this :tada:
